pkg/virtctl/imageupload/imageupload.go

```diff
@@ -926,3 +935,38 @@ func handleEventErrors(client kubecli.KubevirtClient, pvcName, dvName, namespace
 
 	return nil
 }
+
+func createDataSource(client kubecli.KubevirtClient, name, namespace string) error {
+	if ds, _ := client.CdiClient().CdiV1beta1().DataSources(namespace).Get(context.Background(), name, metav1.GetOptions{}); ds != nil {
+		ds.Spec.Source.PVC = &v1beta1.DataVolumeSourcePVC{
+			Name:      name,
+			Namespace: namespace,
+		}
+		client.CdiClient().CdiV1beta1().DataSources(namespace).Update(context.Background(), ds, metav1.UpdateOptions{})
+
+		fmt.Printf("Updated already existing DataSource %s/%s\n", namespace, name)
```

Comment:
```suggestion
		cmd.Printf("Updated already existing DataSource %s/%s\n", namespace, name)
```

Reply:
That should be used everywhere btw...

Reply:
Do you want to import a dependency to `cmd` and change it everywhere, or just this one?